### DISCUSSION TITLE
Sampling within a transect

### DISCUSSION THREAD

**Discussion by PentiumWater** [DHI/mikeio#766] [Q&A]
Is it possible to sample time series data along a transect (i.e., between two known points) from a dfs2 file (specifically flows).  So far, I've been able to extract time series along an axis and at a point, but am hoping to extract data for a number of transects (that don't align with the x,y axes).  Thanks

**Comment by ecomodeller** [comment]
Not at the moment. What format, data structure would you like to get as the result of the extraction?
